content/storage/docs/spark/ebs.md

```diff
@@ -86,7 +104,7 @@ cat >spark-python-in-s3-ebs-static-localdir.json << EOF
   "name": "spark-python-in-s3-ebs-static-localdir", 
   "virtualClusterId": "<virtual-cluster-id>", 
   "executionRoleArn": "<execution-role-arn>", 
-  "releaseLabel": "emr-6.2.0-latest", 
+  "releaseLabel": "emr-6.15.0-latest", 
```

Comment:
Better mention the minimum emr release version to support this feature.

Reply:
mentioned the dynamic PVC is supported for EMR versions >= 6.3.0